rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.84" # ⚠️ Also change in top-level Cargo.toml
+channel = "nightly-2025-04-16" # ⚠️ Also change in top-level Cargo.toml
```

Comment:
Doesn't match Cargo.toml's ?

Reply:
In `Cargo.toml` you can't use the nightly syntax so I used the closest stable version.